lib/matplotlib/axes/_base.py

```diff
@@ -2594,6 +2647,7 @@ def draw(self, renderer=None, inframe=False):
             renderer.stop_rasterizing()
 
         mimage._draw_list_compositing_images(renderer, self, artists)
+        self._update_title_position(renderer)
```

Comment:
Why exactly is it called a second time in `draw` (first call at l. 2598)?

Reply:
Seems to work w/o the second call.  Good catch!